spec/integration/admin_censor_rule_spec.rb

```diff
@@ -0,0 +1,53 @@
+# -*- encoding : utf-8 -*-
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+require File.expand_path(File.dirname(__FILE__) + '/alaveteli_dsl')
+
+describe 'Adding an authority censor rule' do
+  before do
+    allow(AlaveteliConfiguration).to receive(:skip_admin_auth).and_return(false)
+
+    confirm(:admin_user)
+    @admin = login(:admin_user)
+    @authority = FactoryGirl.create(:public_body)
+  end
+
+  it 'clears the cache for existing requests' do
+    raw_email_data = <<-EOF.strip_heredoc
```

Comment:
You could move all of this set up into `let` blocks or the `before`, to make the spec clearer in what it's actually testing vs what is just data for it to run?